doc/modules/postprocessing.rst

```diff
@@ -182,7 +182,7 @@ Other postprocessing tools
 align_sorting
 ^^^^^^^^^^^^^
 
-This function aligns the spike trains a :code:`BaseSorting` object using pre-computed shits of misaligned templates.
+This function aligns the spike trains a :code:`BaseSorting` object using pre-computed shifts of misaligned templates.
```

Comment:
Nice catch ahahahah

Reply:
The number of times I've done this exact same thing gives me nightmares haha